site/layouts/shortcodes/addons.html

```diff
@@ -2,6 +2,7 @@
     <thead  align="left">
     <tr>
         <th style="width: 50%">Name</th>
+        <th>ID</th>
```

Comment:
The other fields set width and total 100, do they all need adjusting or does the browser just figure it out?

Reply:
I did not test all browsers, in Firefox it displays fine.